cypress/integration/dib-travel/company-settings/reference-fields/helpers/delete-cost-center.ts

```diff
@@ -0,0 +1,16 @@
+export const deleteCostCenter = (referenceFields: string) => {
+  cy.waitForAngular();
+
+  cy.get('dib-company-management dib-reference-fields dib-cost-center .table-cell h4')
+    .contains(referenceFields)
+    .parent('.table-cell')
+    .next('.table-cell')
+    .next('.table-cell')
+    .next('.button-cell')
+    .contains(' Archive ')
+    .click();
+
+  cy.get('.cdk-overlay-container confirmation-dialog ui-button[type=warning').click();
+
+  cy.waitForAngular();
```

Comment:
Is `ci.vaitForAngular ()` needed here? Because, I see that there is at the beginning of the test?

Reply:
Removed